.github/workflows/ci-meson.yml

```diff
@@ -168,9 +168,22 @@ jobs:
             pytest --doctest-ignore-import-errors --doctest -rfEs -s src || true
           else
             pytest -rfEs -s src
-            ./sage -t ${{ matrix.tests == 'all' && '--all' || '--new --long' }} -p4 --format github
+            ./sage -t ${{
+              matrix.tests == 'all' &&
+              '--all-except="src/sage/libs/singular/function.pyx src/sage/rings/polynomial/plural.pyx"' ||
```

Comment:
Can you please extract the list of flaky tests to an env variable set before the `steps`, and then reuse this here and in the flaky files test.